components/input/input.css

```diff
@@ -0,0 +1,46 @@
+.form-input{
+    display: flex;
+    flex-direction: column;
+}
+
+.btn{
+    margin-left: 0%;
+    padding: 0.5rem;
+}
+
+.text-label{
+    font-size: 1rem;
+    font-weight: 600;
+}
+
+.input-text{
+    border: 1px solid var(--default-color);
+    border-radius: 4px;
+    background-color: #fff;
```

Comment:
as you are using variables then try to be more consistent with them,
as is looks clean, and easy to maintain.